### PR TITLE
MyBike landing

### PR DESCRIPTION
**Pull request by MaksymMaliuk**
[DEMO LINK](https://MaksymMaliuk.github.io/layout_miami/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,197 @@
...
+             class="logo"
+            >
+              <img
+                src="images/logo.svg"
+                alt="MyBike"
+                class="logo__image"
+              >
+            </a>
+            <div class="top-actions">
+              <a href="tell:+1 234 5555-55-55" class="icon--phone icon header__call"></a>
```
Comment (vladyslav-mosor) [inline src/index.html:28]: 'tell' => 'tel'

**Review by vladyslav-mosor — CHANGES_REQUESTED** [review]
1. Disable scroll for the page when your menu is opened. And it`s better also to close the menu automatically when the page is refreshed during the menu opened, but it goes up to you.

2. Your images must be placed in the center on tablets and phones.

3. Your link that is shown when icon call is hovered does not work as link to phone number. Please fix it

**Review by sergey-lipikhin — CHANGES_REQUESTED** [review]
Change color while hovering on phone, email and address. And with transition for color changing (CL p. 4)
image

image


Text of form fields has different styles?
image

Avoid empty values in form fields, according to checklist
image

**Review by sergey-lipikhin — APPROVED** [review]

**Review by Andr1yK — CHANGES_REQUESTED** [review]
Why I can't submit form?)
image

File: src/styles/blocks/header.scss [hunk 2]
```diff
@@ -0,0 +1,94 @@
+.header {
+  background-image: url(../images/bitmaps/cover.png);
```
Comment (sergii-nosachenko) [inline src/styles/blocks/header.scss:2]: Why do you need so a huge image for bg? PNG is for icons or illustrations, but not for photos. Use JPG instead.

File: src/styles/blocks/footer.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.footer {
+  height: 200px;
+  background: url(../images/bitmaps/footer.png) no-repeat center;
```
Comment (sergii-nosachenko) [inline src/styles/blocks/footer.scss:3]: Same issue with file format

**Review by sergii-nosachenko — APPROVED** [review]
Good work.
Commented on the chosen image format.

**Pull request closed without merging**